openshift-4.0/images/tuned.yml

```diff
@@ -1,11 +1,11 @@
-name: openshift/ose-tuned
+name: openshift/ose-cluster-node-tuned
 
 owners:
 - jmencak@redhat.com
 
 content:
   source:
-    alias: tuned
+    alias: cluster-node-tuned
```

Comment:
alias should not change, should still point at the source alias listed in the group yaml. we're only changing the name (docker repo) the image gets after building.